docs/docs/basics/00-app-anatomy.md

```diff
@@ -61,31 +61,31 @@ https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/simapp/app.go#L151-L193
 
 ### Constructor Function
 
-This function constructs a new application of the type defined in the section above. It must fulfill the `AppCreator` signature in order to be used in the [`start` command](../core/03-node.md#start-command) of the application's daemon command.
+Also defined in `app.go` is the construction function, which constructs a new application of the type defined in the preceding section. The function must fulfill the `AppCreator` signature in order to be used in the [`start` command](../core/03-node.md#start-command) of the application's daemon command.
 
 ```go reference
 https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/server/types/app.go#L57-L59
 ```
 
 Here are the main actions performed by this function:
 
-* Instantiate a new [`codec`](../core/05-encoding.md) and initialize the `codec` of each of the application's module using the [basic manager](../building-modules/01-module-manager.md#basicmanager)
-* Instantiate a new application with a reference to a `baseapp` instance, a codec and all the appropriate store keys.
-* Instantiate all the [`keeper`s](#keeper) defined in the application's `type` using the `NewKeeper` function of each of the application's modules. Note that `keepers` must be instantiated in the correct order, as the `NewKeeper` of one module might require a reference to another module's `keeper`.
+* Instantiate a new [`codec`](../core/05-encoding.md) and initialize the `codec` of each of the application's modules using the [basic manager](../building-modules/01-module-manager.md#basicmanager).
+* Instantiate a new application with a reference to a `baseapp` instance, a codec, and all the appropriate store keys.
+* Instantiate all the [`keeper`](#keeper) objects defined in the application's `type` using the `NewKeeper` function of each of the application's modules. Note that keepers must be instantiated in the correct order, as the `NewKeeper` of one module might require a reference to another module's `keeper`.
```

Comment:
Per [Grammatical treatment of code elements](https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements), I'm attempting to remove inflections after names of code elements. Or, should they remain as-is?

Reply:
@julienrbrt Could you let me know what you think of this?

> @julienrbrt @barriebyron Per [Grammatical treatment of code elements](https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements), I'm attempting to remove inflections after names of code elements, as in `keepers` -> `keeper` objects. Or, should they remain as-is?